### ISSUE TITLE
How to treat etymological dictionaries?

### ISSUE THREAD

**Issue by xrotwang** [lexibank/csd#8]
Currently, CSD isn't really treated seriously. Concepts are mapped only by string equality check, cognates are not considered. So I think we either drop the idea of a CSD lexibank set, or we try to make more sense of it (in particular since it's north american languages!).

So as far as I can tell, CSD is organized not as set of concepts, but as set of etymologies, (see https://csd.clld.org/parameters), with `(1|2|...)` marking different etymologies for the same concept. Then the concept list would only need to map ~1,000 different concepts. Etymologies could be modeled as cognate sets, I think, maybe with some sort of additional flag marking that they may not be exactly "same concept, sound change based cognates".

Whether we can make sense of the transcriptions is unclear to me. If so, adding automatic alignments to the etymologies may be an interesting exercise.

**Comment by xrotwang** [comment]
Re transcriptions: At least for ~3,000 of the ~19,000 lexemes there's a "phonetic siouan" transcription, see https://csd.clld.org/values

**Comment by LinguList** [comment]
We have already shown [here](https://doi.org/10.17613/gscz-mb13) that in theory, the retro-standardization of IGT and similar resources is possible, although the results often do not show high overlap with the concept lists one would look for, or large gaps. 

But I think we should not in principal exclude the possibility of treating these as lexibank datasets, but we would need an expert who could explain the transcription system. Or somebody would have to dig into the specifics of Siouan in order to find it out. 

Transcriptions look also pretty [NAPA](https://clts.clld.org/contributions/napa) to me.

The nice thing is that we have word trees in there (!). There are not many other dictionaries showing such a detail. So I think: once we have the forms, meanings, in cldf, we can give it a try.

**Comment by xrotwang** [comment]
@chrzyki do you want to have a go at the concepticon mapping?

**Comment by LinguList** [comment]
A rule for concepticon mapping here would be: play save. Thus, rather prefer not to link then to link too much. This also makes it easier to link.

**Comment by xrotwang** [comment]
Yes, if we end up with 300 safe matches, that would be already awesome.

**Comment by LinguList** [comment]
Question, @xrotwang, is the formspec up to date? I see a lot of complex things going on there, in the entries.

**Comment by xrotwang** [comment]
It's just the default form spec at work. As I said, this dataset wasn't treated seriously so far.

**Comment by xrotwang** [comment]
I also just realize that the source CLDF from `raw/` doesn't even include the supposed IPA from the `phonetic Siouan` column. So I'll need to recreate the source first.

**Comment by LinguList** [comment]
While it is good to have this, I think the entries are all phonetic enough (with all problems), but I think phonetic here means: as opposed to phonological, since the forms do not differ too much. So we should segment on the "form".

**Comment by LinguList** [comment]
There's one big challenge now for formspec:

* some "splitters" should yield two or more forms for one value, e.g., those with a comma, showing variants 
* some are mere variants, so we can ignore them, like `/` or `~`

similar for brackets

* `[]` is some indicator of irregularity, can be ignored (also in orthoprofile
* `{}` should be stripped off
